### DISCUSSION TITLE
New Release Not Working in Firefox.

### DISCUSSION THREAD

**Discussion by DomDiddly** [aclap-dev/video-downloadhelper#578] [Help and Q&A]
### Your question

Installed version 9.0.1.13 and CoApp one hour ago. I had been using the previous versions of VDH with no problems. 
I selected a video to download, clicked the download button and the icon spins continuously showing 0% downloaded. The download hasn`t started. Previously on that site, the download would be immediate and fast. 

It may be because I`m running an old machine using Windows 7. 

I would really like to be able to revert to the previous version of this great app!

Thanks,
Dom. 

### Addon & CoApp version and details

Installed version 9.0.1.13 and CoApp one hour ago. I had been using the previous versions of VDH with no problems. 
I selected a video to download, clicked the download button and the icon spins continuously showing 0% downloaded. The download hasn`t started. Previously on that site, the download would be immediate and fast. 

It may be because I`m running an old machine using Windows 7. 

I would really like to be able to revert to the previous version of this great app!

Thanks,
Dom.

**Comment by paulrouget** [comment]
In Settings > More Settings, click on "Copy Information To Clipboard" and paste it here. I need to know which version of the CoApp you're using.

> I would really like to be able to revert to the previous version of this great app!

You can always do that. See: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface

**Comment by DomDiddly** [comment]
version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: "checking"
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0
{
    "http_media_download_strategy": "coapp",
    "used_sidebar_button": true,
    "used_popup_button": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 14
}

**Reply by paulrouget** [reply]
Hmm… there's an issue with your CoApp. Can you reinstall it? https://www.downloadhelper.net/install-coapp-v2

**Comment by DomDiddly** [comment]
I noticed that the download was hanging on `verifying Companion App``

**Reply by paulrouget** [reply]
Right. That's the issue. Can you check that your antivirus is not blocking the coapp?
